src/core_scan/Core_scan.ml

```diff
@@ -167,7 +167,7 @@ let replace_named_pipe_by_regular_file path =
         let remove () = if Sys.file_exists tmp_path then Sys.remove tmp_path in
         (* Try to remove temporary file when program exits. *)
         at_exit remove;
-        Fun.protect
+        Common.protect
```

Comment:
probably need to do the same modif in semgrep-pro in deep_scan.ml

Reply:
I don't see any `Fun.protect` there, but I'll open a PR to update Pro.